{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "name" : "categories",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "requests.exceptions.ConnectionError: HTTPConnectionPool(host='calapi.51jirili.comhttp', port=80): Max retries exceeded with url: //calapi.51jirili.com/config/common (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002B1A6976E80>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))",
      "children" : [ {
        "name" : "test_config_common",
        "uid" : "f7c7f2d61624dd4a",
        "parentUid" : "23b7fed1ad71a0a6f8062bb053621eba",
        "status" : "broken",
        "time" : {
          "start" : 1600304444079,
          "stop" : 1600304444094,
          "duration" : 15
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ ]
      } ],
      "uid" : "23b7fed1ad71a0a6f8062bb053621eba"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ]
}